base/cvd/cuttlefish/host/commands/cvd/server_command/status_fetcher.cpp

```diff
@@ -224,10 +224,28 @@ Result<StatusFetcherOutput> StatusFetcher::FetchStatus(
 }
 
 Result<Json::Value> StatusFetcher::FetchGroupStatus(
-    const selector::LocalInstanceGroup& group,
+    selector::LocalInstanceGroup& group,
```

Comment:
nit: now that this is an [input/output parameter](https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs), it should be after the input parameters.